GUI/Controls/AllModVersions.resx

```diff
@@ -118,8 +118,9 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="label1.Text" xml:space="preserve"><value>All available versions of selected mod</value></data>
-  <data name="ModVersion.Text" xml:space="preserve"><value>Mod Version</value></data>
-  <data name="CompatibleGameVersion.Text" xml:space="preserve"><value>Compatible Game Versions</value></data>
+  <data name="ModVersion.Text" xml:space="preserve"><value>Mod version</value></data>
+  <data name="CompatibleGameVersion.Text" xml:space="preserve"><value>Compatible game versions</value></data>
```

Comment:
Should we get rid of the word `Compatible` here, to make the string a bit shorter? It's kind of implied by the fact that it's a mod version->game version mapping, and should be rather obvious from looking at the table.

That would relax the horizontal real estate situation a bit.

Reply:
Done!